.github/workflows/self-new-model-pr-caller.yml

```diff
@@ -0,0 +1,125 @@
+name: New model PR CI
+
+on:
+  pull_request:
+    paths:
+      - "src/transformers/models/**"
+
+env:
+  HF_HOME: /mnt/cache
+  TRANSFORMERS_IS_CI: yes
+  OMP_NUM_THREADS: 8
+  MKL_NUM_THREADS: 8
+  RUN_SLOW: yes
+  # For gated repositories, we still need to agree to share information on the Hub repo. page in order to get access.
+  # This token is created under the bot `hf-transformers-bot`.
+  HF_HUB_READ_TOKEN: ${{ secrets.HF_HUB_READ_TOKEN }}
+  SIGOPT_API_TOKEN: ${{ secrets.SIGOPT_API_TOKEN }}
+  TF_FORCE_GPU_ALLOW_GROWTH: true
+  RUN_PT_TF_CROSS_TESTS: 1
+  CUDA_VISIBLE_DEVICES: 0,1
+
+jobs:
+  check_for_new_model:
+      runs-on: ubuntu-22.04
+      name: Check if PR is a new model PR
+      if: contains(github.event.pull_request.labels.*.name, 'single-model-run-slow')
```

Comment:
After looking into the GitHub Actions doc and online discussion, I decide to add this condition.

**Only** after the PR has been labeled with `single-model-run-slow`, the subsequent `pull_request` events (like push more commits to the PR) will trigger the workflow.

Note that, the **push** after being labeled by `single-model-run-slow` is still necessary to trigger the CI.
I avoid to trigger the CI by using `issue_comment` (as this is way too frequent, despite we can filter it out later).

So please ask the contributors to push an empty commit after add this label. For external contributors' PR, we still need to click `Approve and run` button.


We can explore the `pull_request: labeled` event in the future.

Reply:
> add the label single-model-run-slow
This is the requirement, but it won't actually trigger the CI. The contributor has to push a commit (even empty) to trigger it. 

But we can explore to use another event **together** where when we add a label (`pull_request: labeled`). In that event, the CI will be triggered (if the added label is `single-model-run-slow`).

Why we don't just use the above: because the PR might need multiple CI runs before it is ready to be merged. And we don't want the reviewers to add label / remove it / re-add label  to achieves this.